students/nartowska_karina/lesson_05_lists/chess_board.py

```diff
@@ -0,0 +1,34 @@
+def chess_board(rows, columns):
+    board = []
+    for i in range(rows):
+        row = []
+        for j in range(columns):
+            if i % 2 != j % 2:
+                row.append("*")
+            else:
+                row.append(".")
+        board.append(row)
+    print("Result:")
```

Comment:
Same as previously, better is to print this in main() where is also printed result.